app/src/main/java/com/group1/movebetter/nextbike/MapFragment.kt

```diff
@@ -26,32 +34,36 @@ import com.mapbox.mapboxsdk.maps.MapView
 import com.mapbox.mapboxsdk.maps.MapboxMap
 import com.mapbox.mapboxsdk.maps.OnMapReadyCallback
 import com.mapbox.mapboxsdk.maps.Style
-import com.mapbox.mapboxsdk.style.layers.SymbolLayer
-import com.mapbox.mapboxsdk.plugins.annotation.OnSymbolClickListener
-import com.mapbox.mapboxsdk.plugins.annotation.Symbol
-import com.mapbox.mapboxsdk.plugins.annotation.SymbolManager
-import com.mapbox.mapboxsdk.style.layers.Property.ICON_ANCHOR_BOTTOM
+import com.mapbox.mapboxsdk.style.expressions.Expression.*
 import com.mapbox.mapboxsdk.style.layers.PropertyFactory.*
+import com.mapbox.mapboxsdk.style.layers.SymbolLayer
+import com.mapbox.mapboxsdk.style.sources.GeoJsonSource
 
-class MapFragment : Fragment(), OnMapReadyCallback, PermissionsListener {
 
-    private val CALLOUT_LAYER_ID = "CALLOUT_LAYER_ID"
-    private val GEOJSON_SOURCE_ID = "GEOJSON_SOURCE_ID"
+class MapFragment : Fragment(), OnMapReadyCallback, PermissionsListener, MapboxMap.OnMapClickListener {
+
+    private val BIKE_STATION_LAYER = "bike-station-layer"
+    private val BIKE_STATIONS = "bike-stations"
+    private val BIKE_NETWORK_LAYER = "bike-network-layer"
+    private val BIKE_NETWORK_SOURCE = "bike-network-source"
+    private val SELECTED_MARKER = "selected-marker"
+    private val SELECTED_MARKER_LAYER = "selected-marker-layer"
```

Comment:
Die Strings auslagern in res/values/strings.xml

Reply:
Erledigt